lib/viget/deployment/shared/configuration.rb

```diff
@@ -17,19 +31,21 @@ def remote_file_exists?(path)
         results.all?
       end
 
-      def find_templates_in(path)
-        @found_templates ||= Dir.glob("#{path}/*.erb").inject({}) do |mapping, path|
-          mapping.merge!(path => File.basename(path, '.erb'))
+      def find_templates_in(*paths)
+        pattern = (paths.length > 1) ? "{#{paths.join(',')}}/*.erb" : "#{paths.first}/*.erb"
```

Comment:
Since something like `"{foo}/*.erb"` will be an empty set even if there are `.erb` files in `foo` due to how globbing seems to work with  `{}` .